<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <testsuite name="force.apex" timestamp="2021-04-11T18:57:54+02:00" hostname="https://data-page-19352-dev-ed.cs101.my.salesforce.com" tests="1" failures="1"  errors="0"  time="1.43">
        <properties>
            <property name="outcome" value="Failed"/>
            <property name="testsRan" value="1"/>
            <property name="passing" value="0"/>
            <property name="failing" value="1"/>
            <property name="skipped" value="0"/>
            <property name="passRate" value="0%"/>
            <property name="failRate" value="100%"/>
            <property name="testStartTime" value="Apr 11, 2021 6:57 PM"/>
            <property name="testExecutionTime" value="1.43 s"/>
            <property name="testTotalTime" value="1.43 s"/>
            <property name="commandTime" value="4.07 s"/>
            <property name="hostname" value="https://data-page-19352-dev-ed.cs101.my.salesforce.com"/>
            <property name="orgId" value="00D1X00000034R1UAI"/>
            <property name="username" value="test-in7nqa4fsmgt@example.com"/>
            <property name="testRunId" value="7071X00001WrSqH"/>
            <property name="userId" value="0051X000007SSFeQAO"/>
        </properties>
        <testcase name="should_aggregate_two_orders_on_account" classname="TestUpdateAccountTurnover" time="1.43">
            <failure message="System.SObjectException: SObject row was retrieved via SOQL without querying the requested field: Order.AccountId"><![CDATA[Class.TestUpdateAccountTurnover.should_aggregate_two_orders_on_account: line 89, column 1]]></failure>
        </testcase>
    </testsuite>
</testsuites>
